saleor/graphql/account/mutations.py

```diff
@@ -453,10 +455,38 @@ def user_is_allowed(cls, user, input):
         return user.has_perm('account.manage_users')
 
 
+class CustomerAddressCreate(ModelMutation):
+    class Arguments:
+        input = AddressInput(
+            description='Fields required to create address', required=True)
+
+    class Meta:
+        description = 'Creates address for user'
+        model = models.Address
+        exclude = ['user_addresses']
+
+    @classmethod
+    def user_is_allowed(cls, user, input):
+        return user.is_authenticated
+
+    @classmethod
+    def save(cls, info, instance, cleaned_input):
+        user = info.context.user
+        super().save(info, instance, cleaned_input)
```

Comment:
It would make more sense to put the user variable after the super call. Or even fully remove it as it is redundant.
Unless I'm missing something out.

Reply:
Why? I don't see an issue here. We're saving the address instance and assigning it to the user. Only the `instance.save()` below seems superficial, `add()` already saves data in the database.

Reply:
The reason that I'm thinking is that the super call is being done after getting the user from the context, which by reading the code line per line makes it seem needed by the super call.
Then, for the second point, it seems that doing `instance.user_addresses.add(info.context.user)` would be pretty straightforward, as the user directly see that we are passing the user. So I suggested that maybe we could simplify to that.